components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -419,10 +424,14 @@ public boolean deleteBusinessRule(String uuid, Boolean forceDeleteEnabled) {
             boolean isCompletelyUndeployed = true;
 
             for (String nodeURL : nodeList) {
-                isCompletelyUndeployed = undeploySiddhiApp(nodeURL, foundBusinessRuleFromScratch.getUuid());
+                isCompletelyUndeployed = undeploySiddhiApp(nodeURL, businessRuleFromScratch.getUuid());
+                if (!isCompletelyUndeployed) {
+                    log.error("");
```

Comment:
What is the error message?

Reply:
Reimplemented by 2c0119c4c48e1077512e685493252c388e68a520